Tactics/Sym.lean

```diff
@@ -23,6 +23,12 @@ private def evalTacticAndTrace (tactic : TSyntax `tactic) : TacticM Unit :=
     evalTactic tactic
     trace[Tactic.sym] "new goal state:\n{← getGoals}"
 
+private def traceHeartbeats (header : Option String := none) : MetaM Unit := do
```

Comment:
This could belong in a common file for use by all the tactics we're developing.

Reply:
Done, I moved it to `Tactics.common`